### PR TITLE
fix(refs dplan-12823): Reset orga form only for editable fields

### PR DESCRIPTION
**Pull request by salisdemos**
### Ticket
DPLAN-12823

On Orga-Edit page (e.g. as FPA)
`/organisation/edit/<orga-id>`

When resetting the page, non-editable should not change (not be cleared)
Other Fields should change to the initial value (not be cleared if there was initial content)

In addition I moved some code. Sorry. That should have been a separate PR. If wanted I can guide through the changes.
The Commits are quite clear to follow.

### CODE REVIEW

File: client/js/components/user/orgaDataEntry/OrganisationDataForm.vue [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
         {{ Translator.trans('organisation.data') }}
       </legend>
 
         <!-- Name -->
         <dp-input
           id="orga_name"
           v-model="organisation.name"
-          class="mb-2"
+          :class="prefixClass('mb-2')"
           data-cy="organisationData:name"
```
Comment (gruenbergerdemos) [inline client/js/components/user/orgaDataEntry/OrganisationDataForm.vue:32]: Do we still need these prefixClasses?

File: client/js/components/user/orgaDataEntry/OrganisationDataForm.vue [hunk 2]
```diff
@@ -322,16 +353,47 @@ export default {
...
+        || this.hasNotificationSection
+        || this.showDetailedInfo
+        || hasPermission('feature_change_submission_type')
     },
-
-    displayCustomer () {
-      return hasPermission('feature_display_customer_names') && this.customers?.length
+  },
+  methods: {
+    precheckSubmit (e) {
```
Comment (gruenbergerdemos) [inline client/js/components/user/orgaDataEntry/OrganisationDataForm.vue:378]: This seems to not get called on submit (tested locally)

**Review by gruenbergerdemos — CHANGES_REQUESTED** [review]
Nice refactoring, thanks :)
Just a general question and the pre submit check seems not be working

File: client/js/components/user/orgaDataEntry/OrganisationDataForm.vue (hunk 1 above)
Comment (salisdemos) [reply client/js/components/user/orgaDataEntry/OrganisationDataForm.vue:32]: Yes. we should keep that functionality. ;-)

File: client/js/components/user/orgaDataEntry/OrganisationDataForm.vue (hunk 2 above)
Comment (salisdemos) [reply client/js/components/user/orgaDataEntry/OrganisationDataForm.vue:378]: Not at all? Or did you just don't run into the if?

Comment (gruenbergerdemos) [reply client/js/components/user/orgaDataEntry/OrganisationDataForm.vue:378]: I set the breakpoint before the if and it didn't stop

Comment (hwiem) [reply client/js/components/user/orgaDataEntry/OrganisationDataForm.vue]: Fixed in c18f3b966f51b0ad650dc9b53d3c387f8d0bc088

**Review by gruenbergerdemos — APPROVED** [review]
lgtm, thanks for the fix

**Pull request merged**